### PR TITLE
SparkSubmit: Adding propertyfiles option

### LINKED ISSUE #34838: Adding property files option in the Spark Submit command
### Description

spark-submit command has one of the options to pass the properties file as argument. Instead of loading multiple key, value via --conf option, this will help to load extra properties from the file path. While we have support for most of the arguments supported in spark-submit command in SparkSubmitOperator, this one `property-files` is missing. Could that be included?
```[root@airflow ~]# spark-submit --help
Usage: spark-submit [options] <app jar | python file | R file> [app arguments]
Usage: spark-submit --kill [submission ID] --master [spark://...]
Usage: spark-submit --status [submission ID] --master [spark://...]
Usage: spark-submit run-example [options] example-class [example args]

Options:

 --conf, -c PROP=VALUE       Arbitrary Spark configuration property.
  --properties-file FILE      Path to a file from which to load extra properties. If not
                              specified, this will look for conf/spark-defaults.conf.
```

### Use case/motivation

Add the property-files as one of the options to pass in the SparkSubmitOperator to load the extra config properties as file

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by pateash**
closes: #34838

## Description
There is an option to override the default conf/spark-defaults.conf properties in Spark-submit
image

We want to have that option in SparkSubmitOperator and Hook as well.

### CODE REVIEW

**Comment by pateash** [comment]
cc. @ghostp13409

File: tests/system/providers/apache/hive/example_twitter_README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
 1. Save Summary to MySQL
 
 ***Screenshot:***
-<img src="http://i.imgur.com/rRpSO12.png" width="99%"/>
+<img src="http://i.imgur.com/rRpSO12.png" width="99%" alt="Dag for Collecting tweets for four users account"/>
```
Comment (hussein-awala) [inline tests/system/providers/apache/hive/example_twitter_README.md]: Is this change related to the main goal of this PR?

Comment (pateash) [reply tests/system/providers/apache/hive/example_twitter_README.md]: No, this was a pre-commit failure.
as all the images need an alt-text, may be due to addition of this new rule
https://github.com/DavidAnson/markdownlint/blob/main/doc/md045.md

**Comment by bolkedebruin** [comment]
Can you add tests please?

File: tests/system/providers/apache/hive/example_twitter_README.md (hunk 1 above)
Comment (eladkal) [reply tests/system/providers/apache/hive/example_twitter_README.md]: Fixed in https://github.com/apache/airflow/pull/36169

**Review by potiuk — APPROVED** [review]
